### PR TITLE
Resolves #2063

### LINKED ISSUE #2063: Please add more examples for the ssl case
Hi everybody. 

Recently I was working to deploy automatically a few sites, when I was reading the examples is only one example and it works pretty well, but we can add more examples for website that are running under https protocol. 

Thank you very much, 
Sincerely, 
H

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 95401782-16b2-2fa5-7146-7051c0ac5163
* Version Independent ID: 1f7fd84b-0c0c-5954-8666-18c3dec674e4
* Content: [New-WebBinding (webadminstration)](https://docs.microsoft.com/en-us/powershell/module/webadminstration/new-webbinding?view=winserver2012-ps&redirectedfrom=MSDN&viewFallbackFrom=winserver2012r2-ps)
* Content Source: [docset/winserver2012-ps/webadminstration/New-WebBinding.md](https://github.com/MicrosoftDocs/windows-powershell-docs/blob/master/docset/winserver2012-ps/webadminstration/New-WebBinding.md)
* Product: **w10**
* Technology: **powershell**
* GitHub Login: @andreabarr
* Microsoft Alias: **v-anbarr**

### PR DESCRIPTION
**Pull request by velkovb**
Add ssl example.
https://github.com/MicrosoftDocs/windows-powershell-docs/issues/2063

### CODE REVIEW

**Comment by e0i** [comment]
@dariomws @get-itips @tseward

Peer review needed for this PR.

cc @JohanFreelancer9 for copy editing.

Thank you.

File: docset/winserver2012-ps/webadminstration/New-WebBinding.md [hunk 1]
```diff
@@ -22,16 +22,23 @@ New-WebBinding [[-Name] <String>] [-Protocol <String>] [-Port <UInt32>] [-IPAddr
...
+
+This command creates a binding on the default website.
+
+### Example 2: Add a new ssl site binding
+```powershell
+IIS:\> New-WebBinding -Name "Default Web Site" -IPAddress "*" -Port 443 -HostHeader "TestSite" -Protocol "https"
 ```
 
-Creates a new binding on the Default Web Site.
+This command creates an SSL binding on the default website.s
```
Comment (dariomws) [inline docset/winserver2012-ps/webadminstration/New-WebBinding.md:41]:
```suggestion
This command creates an SSL binding on the default website.
```

**Review by dariomws — CHANGES_REQUESTED** [review]

**Review by get-itips — COMMENTED** [review]
i am ok once suggestions are applied

**Comment by tseward** [comment]
Also OK once the changes are applied.

**Review by dariomws — APPROVED** [review]

**Review by get-itips — APPROVED** [review]

**Comment by tseward** [comment]
Actually, revisiting this -- the cmdlet examples don't show how to get into the IIS drive. Should we instead just use PS C:\> for the examples?

File: docset/winserver2012r2-ps/webadministration/New-WebBinding.md [hunk 2]
```diff
@@ -34,13 +34,20 @@ The **New-WebBinding** cmdlet adds a new binding to an existing website.
...
-```
-IIS:\>New-WebBinding -Name "Default Web Site" -IPAddress "*" -Port 80 -HostHeader "TestSite"
+### Example 1: Add a new site binding
+```powershell
+IIS:\> New-WebBinding -Name "Default Web Site" -IPAddress "*" -Port 80 -HostHeader "TestSite"
 ```
 
 This command creates a binding on the default website.
 
+### Example 2: Add a new ssl site binding
```
Comment (JohanFreelancer9) [inline docset/winserver2012r2-ps/webadministration/New-WebBinding.md:44]:
```suggestion
### Example 2: Add a new SSL site binding
```

File: docset/windows/webadministration/New-WebBinding.md [hunk 3]
```diff
@@ -36,12 +36,19 @@ The **New-WebBinding** cmdlet adds a new binding to an existing website.
...
 
 ### Example 1: Add a new site binding
-```
+```powershell
 IIS:\> New-WebBinding -Name "Default Web Site" -IPAddress "*" -Port 80 -HostHeader "TestSite"
 ```
 
 This command creates a binding on the default website.
 
+### Example 2: Add a new ssl site binding
```
Comment (JohanFreelancer9) [inline docset/windows/webadministration/New-WebBinding.md:45]:
```suggestion
### Example 2: Add a new SSL site binding
```

File: docset/winserver2012-ps/webadminstration/New-WebBinding.md [hunk 4]
```diff
@@ -22,16 +22,23 @@ New-WebBinding [[-Name] <String>] [-Protocol <String>] [-Port <UInt32>] [-IPAddr
...
-### -------------- EXAMPLE 1: Adding a new Site Binding --------------
+### Example 1: Add a new site binding
+```powershell
+IIS:\> New-WebBinding -Name "Default Web Site" -IPAddress "*" -Port 80 -HostHeader "TestSite"
 ```
-IIS:\>New-WebBinding -Name "Default Web Site" -IPAddress "*" -Port 80 -HostHeader TestSite
+
+This command creates a binding on the default website.
+
+### Example 2: Add a new ssl site binding
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/webadminstration/New-WebBinding.md:36]:
```suggestion
### Example 2: Add a new SSL site binding
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - please also see Trevor's comment.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**